<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Yk</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_8">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Угол (гр.)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Нарисовать прямую</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Выберите алгоритмы</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Длина </string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Хk</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Yo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Xo</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Нарисовать пучок</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Цифровой дифференциальный анализатор</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Брезенхема (действительные числа)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Брезенхема (целые числа)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Брезенхема с устранением ступенчатости</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Стандартный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алгоритм Ву</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
